pkg/cluster/internal/kubeadm/config.go

```diff
@@ -440,6 +446,12 @@ conntrack:
 # Skip setting sysctl value "net.netfilter.nf_conntrack_max"
 # It is a global variable that affects other namespaces
   maxPerCore: 0
+# Set sysctl value "net.netfilter.nf_conntrack_tcp_be_liberal"
+# for nftables proxy (theoretically for kernels older than 6.1)
+# xref: https://github.com/kubernetes/kubernetes/issues/117924
+{{if and (eq .KubeProxyMode "nftables") (not .RootlessProvider)}}
+  tcpBeLiberal: true
+{{end}}
```

Comment:
Don't restrict it to nftables mode; if you use `tcpBeLiberal` with iptables kube-proxy then it will skip the "drop ctstate invalid packets" hack, which is a win because that hack breaks other things.

(You _could_ check the kernel version before setting it though...)

Reply:
this option is only available in latest version, and the things that break are with more complex network setups that in kind are not available by default ... I doubt anybody will hit this bug in kind

Reply:
I mean, kind supports the same versions of kubernetes, but there is people using it with very old kubernetes versions

Reply:
ah, if you have to add k8s version checks too that makes it too complicated. ok
/hold cancel